sdccc/src/main/java/com/draeger/medical/sdccc/tests/biceps/invariant/InvariantParticipantModelAnnexTest.java

```diff
@@ -66,6 +69,8 @@ void setup() {
     @TestDescription("Starting from the initially retrieved mdib, applies every episodic report to the mdib and"
             + " verifies that the OperationTarget of every AbstractOperationDescriptor is at any point set to the"
             + " descriptor of the parent of the sco or any child descriptor of that parent.")
+    @RequirePrecondition(
+            manipulationPreconditions = {ManipulationPreconditions.RemoveAndReinsertDescriptorManipulation.class})
```

Comment:
This precondition does not necessarily lead to relevant test data, as the manipulation `RemoveAndReinsertDescriptorManipulation` removes and reinserts a random modifiable descriptor, which does not necessarily have to be an `AbstractOperationDescriptor`. 

One possible idea would be to define a conditional precondition that expects the existence of an
`AbstractOperationDescriptor` and calls a manipulation that inserts one if none exists.

Reply:
> This precondition does not necessarily lead to relevant test data, as the manipulation RemoveAndReinsertDescriptorManipulation removes and reinserts a random modifiable descriptor, which does not necessarily have to be an AbstractOperationDescriptor.

It is supposed to do that for all that support it, not for randomly chosen ones.

However I do agree that this manipulation does not necessarily help in this case, similiarly to what I meant for another case i.e. at https://github.com/Draegerwerk/SDCcc/pull/152#discussion_r1553454611 .

There seem to be more such problematic cases which are are not commented yet, I think the general idea of this PR has to be clarified and then another solution to be found. This PR seems to be in need of a major rework, because more specific manipulations are needed in lots of cases or maybe even all of the cases (, not quite sure yet which one it is).